bedevere/mailer.py

```diff
@@ -0,0 +1,67 @@
+import email.message
+import email.utils
+import os
+
+from gidgethub import routing
+
+
+router = routing.Router()
+
+ALLOWED_BRANCHES = ["2.7", "3.5", "3.6", "master"]
+SENDER = os.environ.get("SENDER_EMAIL", "sender@example.com")
+RECIPIENT = os.environ.get("RECIPIENT_EMAIL", "recipient@example.com")
+
+
+def get_diff_stat(commit):
+    files = {
+        "A": commit["added"],
+        "D": commit["removed"],
+        "M": commit["modified"],
+    }
+    result = []
+    for key, file_list in files.items():
+        if file_list:
+            result.append("\n".join(f"{key} {f}" for f in file_list))
+    return "\n".join(result)
+
+
+def build_message(commit, **kwargs):
+    branch = kwargs.get("branch")
+    diff_stat = kwargs.get("diff_stat")
+    unified_diff = kwargs.get("unified_diff")
+    template = f"""\
```

Comment:
Huh, weird. I don't see how f-strings would be an issue since it is just constructing a string. Another option is to put the string template at the top of the module and use `str.format()`.